tests/tests_unit/test_utils/test_text.py

```diff
@@ -21,6 +21,22 @@ def test_to_camel(self, actual: str, expected: str) -> None:
         assert to_camel_case(actual) == expected
 
 
+class TestToWords:
+    @pytest.mark.parametrize(
+        "actual, expected",
```

Comment:
Nit: actual sounds like an output to be compared to expected, while this is an input